src/app/app.module.ts

```diff
@@ -28,7 +27,6 @@ import { RegistrationComponent } from './Pages/registration/registration.compone
     LoginComponent,
     CartComponent,
     FrontpageComponent,
-    AnimateComponent,
```

Comment:
Removed Component because I couldn't find the component anywhere and it was throwing an error